### DISCUSSION TITLE
[@types/plotly] missing orientation in ColorBar

### DISCUSSION THREAD

**Discussion by gjimenezUCM** [DefinitelyTyped/DefinitelyTyped#69844] [Issues with a @types package]
ColorBar interface [types/plotly.js/index.d.ts#L1493](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/60fa1f7f9f94e140e9b490f5d22fa4c9b17b9ed0/types/plotly.js/index.d.ts#L1493) does not contain the "orientation" attribute (see <https://plotly.com/javascript/reference/heatmap/#heatmap-colorbar> for example. 

This interface should include the following attribute definition:

```
orientation: "v" | "h";
```

[a comment by an automated account is not shown]

**Comment by rodas96** [comment]
any news?

**Comment by hkleungai** [comment]
I have opened PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/73042 for your raised issue. You may want to take a look.
